starboard/testing/fake_graphics_context_provider.cc

```diff
@@ -158,7 +178,23 @@ void FakeGraphicsContextProvider::InitializeWindow() {
 }
 
 void FakeGraphicsContextProvider::InitializeEGL() {
-  display_ = EGL_CALL_SIMPLE(eglGetDisplay(EGL_DEFAULT_DISPLAY));
+  std::vector<const char*> enabled_features = {nullptr};
+  std::vector<const char*> disabled_features = {nullptr};
+  std::vector<SbEglAttrib> display_attribs = {
+      EGL_PLATFORM_ANGLE_DEVICE_TYPE_ANGLE,
+      EGL_PLATFORM_ANGLE_DEVICE_TYPE_EGL_ANGLE,
+      EGL_PLATFORM_ANGLE_TYPE_ANGLE,
+      EGL_PLATFORM_ANGLE_TYPE_OPENGLES_ANGLE,
+      EGL_FEATURE_OVERRIDES_ENABLED_ANGLE,
+      reinterpret_cast<SbEglAttrib>(enabled_features.data()),
+      EGL_FEATURE_OVERRIDES_DISABLED_ANGLE,
+      reinterpret_cast<SbEglAttrib>(disabled_features.data()),
```

Comment:
I think since we don't have any overrides, we can just remove these two altogether like e.g. [here](https://source.chromium.org/chromium/chromium/src/+/main:ui/gl/gl_display.cc;l=205-214;drc=dafa5e3a665408b5d24a449d63297f8694bfea9b) where the EGL_FEATURE_OVERRIDES... are only added when there's something in the vectors. The associated `#define` in  in l.75-79 should be unnecessary as well.